### PR TITLE
removed-punycode

### LINKED ISSUE #919: bug: fix error screaming in the terminal
Screenshot 2024-08-20 at 10 24 02 AM

### PR DESCRIPTION
**Pull request by Ashutoshpadhi629**
#### removes punycode error from the terminal while build

Resolves #919 

###### removed punycode dependecies from package-lock.json 

Screenshot 2024-08-21 at 12 27 46 AM

### CODE REVIEW

File: tsconfig.json
Comment (iamshm) [inline tsconfig.json:1]: this file got added just because of different formatting. You can remove this file

Comment (Ashutoshpadhi629) [reply tsconfig.json:1]: Removed , Thanks

Comment (iamshm) [reply tsconfig.json:1]: I think you still got that wrong
Because you deleted the file
You just need to discard the changes you did

Comment (Ashutoshpadhi629) [reply tsconfig.json:1]: My bad, I deleted the file in hurry.

**Pull request closed without merging**